acceptance/bundle/python/restricted-execution/mutators.py

```diff
@@ -0,0 +1,13 @@
+from databricks.bundles.jobs import Job
+from databricks.bundles.core import job_mutator, Bundle
+import os
+
+
+@job_mutator
+def read_envs(bundle: Bundle, job: Job) -> Job:
+    # This test ensures environment variables are not readable by PyDABs in restriction execution mode
+    # because PyDABs mutators are not executed in the first place
```

Comment:
This could be clearer; the file below does not exist because the code is not executed.